### PR TITLE
refactoring tests

### PR DESCRIPTION
**Pull request by ondrajodas**
Jira - https://keboola.atlassian.net/browse/COM-614

### CODE REVIEW

**Comment by webrouse** [comment]
hned rano sa do toho pustim

File: tests/functional/DatadirTest.php [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+        $sslCa = $config['parameters']['db']['ssl']['ca'] ?? null;
+        $sslCert = $config['parameters']['db']['ssl']['cert'] ?? null;
+        $sslKey = $config['parameters']['db']['ssl']['key'] ?? null;
+
+        if ($sslCa) {
+            $config['parameters']['db']['ssl']['ca'] = $this->getCert($sslCa);
+        }
+        if ($sslCert) {
+            $config['parameters']['db']['ssl']['cert'] = $this->getCert($sslCert);
```
Comment (webrouse) [inline tests/functional/DatadirTest.php:59]: Tu by bolo IMHO lepsie ist tiez cestou `putenv`, boli by to 3 nove riadky v `setUpBeforeClass`.

File: tests/functional/DatadirTest.php [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        parent::setUp();
+
+        // Test dir, eg. "/code/tests/functional/full-load-ok"
+        $this->testProjectDir = $this->getTestFileDir() . '/' . $this->dataName();
+        $this->testTempDir = $this->temp->getTmpFolder();
+
+        $configContent = file_get_contents($this->testProjectDir . '/source/data/config.json');
+
+        $config = json_decode((string) $configContent, true);
+        preg_match('/%env\(string:([A-Z_]+)\)%/', $config['parameters']['db']['host'], $hostEnv);
```
Comment (webrouse) [inline tests/functional/DatadirTest.php:79]: Toto sa moze zmazat, nepouzivaju sa vysledky nikde.

File: tests/functional/error-run-action-without-database/source/data/config.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        "enabled": true
+      }
+    ],
+    "db": {
+      "host": "%env(string:MYSQL_DB_HOST)%",
+      "port": "%env(string:MYSQL_DB_PORT)%",
+      "user": "%env(string:MYSQL_DB_USER)%",
+      "#password": "%env(string:MYSQL_DB_PASSWORD)%"
+    },
+    "extractor_class": "MYSQL"
```
Comment (webrouse) [inline tests/functional/error-run-action-without-database/source/data/config.json:20]: `extractor_class` by tam nemala byt potrebna, nie?

**Review by webrouse — COMMENTED** [review]
@ondrajodas :+1:  len nejake drobnosti -> otazky

File: tests/functional/DatadirTest.php (hunk 1 above)
Comment (ondrajodas) [reply tests/functional/DatadirTest.php]: jj tady se používají jen jedny certifikáty, tak to může být

Comment (ondrajodas) [reply tests/functional/DatadirTest.php]: https://github.com/keboola/db-extractor-mysql/pull/134/commits/14c5555148bac446aa3a6a1e7c1977494bb3f9cd

File: tests/functional/DatadirTest.php (hunk 2 above)
Comment (ondrajodas) [reply tests/functional/DatadirTest.php]: https://github.com/keboola/db-extractor-mysql/pull/134/commits/5b0812e50a1bd4eb3f933414fe3db64f927c739d

File: tests/functional/error-run-action-without-database/source/data/config.json (hunk 3 above)
Comment (ondrajodas) [reply tests/functional/error-run-action-without-database/source/data/config.json]: https://github.com/keboola/db-extractor-mysql/pull/134/commits/6ee9f335f6e39fb4575de618b1a4db7d5caae88e

**Review by webrouse — APPROVED** [review]

**Pull request merged**